virttest/cpu.py

```diff
@@ -736,7 +736,12 @@ def get_cpu_info(session=None):
             logging.info("output is %s" % output)
         finally:
             session.close()
-    cpu_info = dict(map(lambda x: [i.strip() for i in x.split(":", 1)], output))
+    for line in output:
```

Comment:
@kamilvarga , the problem to fix in this PR seems same as https://github.com/avocado-framework/avocado-vt/pull/2987 which was already merged. Could you help verify the problem was already fixed in your test without this PR?

Reply:
Checked with fix from #2987, which works and this PR is therefore no more needed.